libfwupdplugin/fu-firmware.c

```diff
@@ -906,6 +906,11 @@ fu_firmware_parse_full(FuFirmware *self,
 	}
 	if (!fu_firmware_check_magic_for_offset(self, fw, &offset, flags, error))
 		return FALSE;
+
+	/* always set by default */
+	fu_firmware_set_bytes(self, fw);
```

Comment:
if you're always setting this, I think you should also move the alignment check to always check it.

Reply:
I don't think we can, think ihex, srec etc

Reply:
Oh right.